spec/forms/admin/adjustments/create_adjustment_form_spec.rb

```diff
@@ -0,0 +1,107 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe Admin::Adjustments::CreateAdjustmentForm, type: :model do
+  subject(:form) { described_class.new(session:, statement:, description:, amount:) }
+
+  let(:session) { {} }
+  let(:statement) { create(:statement) }
+  let(:description) { nil }
+  let(:amount) { nil }
+
+  describe "#save" do
+    context "when the adjustment is valid" do
+      let(:description) { "Adjustment description" }
+      let(:amount) { 100 }
+
+      it "saves the adjustment" do
+        expect { form.save }.to change(statement.adjustments, :count).by(1)
+      end
+
+      it "adds the adjustment ID to the session" do
+        form.save # rubocop:disable Rails/SaveBang
```

Comment:
Why not just use `save!` - if its invalid it'll just error out the line? and the user can still see the spec thats failing?

Reply:
`save` was a method I'd implemented on the form.
I've renamed it to `save_form` to make it clear it's not a standard activerecord class.